rustls/src/crypto/hpke.rs

```diff
@@ -39,32 +39,73 @@ pub struct HpkeSuite {
 
 /// An HPKE instance that can be used for base-mode single-shot encryption and decryption.
 pub trait Hpke: Debug + Send + Sync {
-    /// Seal the provided `plaintext` to the recipient public key `pk_r` with application supplied
+    /// Seal the provided `plaintext` to the recipient public key `pub_key` with application supplied
     /// `info`, and additional data `aad`.
     ///
     /// Returns ciphertext that can be used with [Self::open] by the recipient to recover plaintext
-    /// using the same `info` and `aad` and the private key corresponding to `pk_r`.
+    /// using the same `info` and `aad` and the private key corresponding to `pub_key`. RFC 9180
+    /// refers to `pub_key` as `pkR`.
     fn seal(
         &mut self,
-        pk_r: &HpkePublicKey,
```

Comment:
It doesn't matter much, but personally I would be in favour of continuing with the argument order from the RFC.

Reply:
It wouldn't be my preference but I also don't feel very strongly about it.

Reply:
I guess I'm the tie breaker :laughing: Like you two my preference isn't strong but I think I lightly prefer arranging the API argument order to match Rustls style vs matching the RFC in this case.